jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -330,6 +342,9 @@
                 export JENKINS_PUBLISHED_VERSION="ci/latest-1.2"
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Ingress\]"
                 export PROJECT="kubernetes-ingress-1-2"
+                # build-1.2 is not using the kubernetes-release-dev bucket yet
```

Comment:
That's my bad. I changed this back to a 1.2 job when I added the 1.3 job but I didn't see that they were broken by release.